tests/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -675,3 +675,36 @@
                   ...
 
               StrCallable = BaseManager.from_queryset(ModelQuerySet, class_name=str(1))
+
+-   case: test_queryset_arg_as_unsupported_expressions
+    main: |
+        from myapp.models import ManagerFromAlias, ManagerFromNonQSGeneric
+        reveal_type(ManagerFromAlias)
+        reveal_type(ManagerFromNonQSGeneric)
+    out: |
+        main:2: note: Revealed type is "Type[django.db.models.manager.Manager[django.db.models.base.Model]]"
```

Comment:
Shouldn't it be `Any`? Usually when some type error happens, we are not sure about the result and set it to `Any.from_error`.

Reply:
Yeah, probably. I think this just returns the default return type right now.

Reply:
Hm, thinking twice about this. Should it really? Our plugin has bailed here and this is just mypy deciding to use the return type from the signature.

So what is happening here shouldn't be anything other than default mypy behaviour.